src/pickup.c

```diff
@@ -3454,6 +3456,29 @@ use_container(
     return used;
 }
 
+static boolean
+transfer_container_available(void)
+{
+    struct obj *otmp, *objchns[2] = { g.invent, g.level.objects[u.ux][u.uy] };
```

Comment:
Should this be updated by itself, or should `select_transfer_container` be updated too (so that it will only list containers in inventory if you're levitating, etc)?